### PR TITLE
Added main script file (+ small modification of configHandler)

### PR DESCRIPTION
**Pull request by Andreas-Menzel**
# I added a main script file (`betawatch.py`) with stuff like argparse

### Start the GUI
Starting the script with no arguments will open the GUI.

### Modify the config file
You can use `--config` to modify the config file from within the terminal.
Currently saved values will be shown when asked for new values. Not entering a new value will keep the old one.

### Use temporary values to open Alfaview
Supplying `--username`, `--password` **or** `--displayName` will start Alfaview with the values provided. Missing values will be taken from the config file or asked for if necessary.

## TODO

- Start the GUI
- Open Alfaview

### CODE REVIEW

File: code/betawatch.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        config['myPassword'] = args.password
+    if args.displayName != None:
+        config['myName'] = args.displayName
+
+    if config['myRoom'] == None:
+        config['myRoom'] = input('What room do you want to join?: ')
+    if config['myUsername'] == None:
+        config['myUsername'] = input('Your HFU account username: ')
+    if config['myPassword'] == None:
+        config['myPassword'] = getpass('Your HFU account password: ')
```
Comment (tizianerlenberg) [inline code/betawatch.py]: Could you output the username, for which the password needs to be entered? Something like: "Your HFU account password for username ..."

File: code/betawatch.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        editConfig()
+
+    if not args.noUI and args.room == None and args.username == None and args.password == None and args.displayName == None and not args.config:
+        startGUI()
+
+    config = configHandler.getConfig()
+
+    if args.room != None:
+        config['myRoom'] = args.room
+    if args.username != None:
```
Comment (tizianerlenberg) [inline code/betawatch.py:106]: Could you please make it, so one also has to enter a new password, every time they set a temporary username with "--username"

**Review by tizianerlenberg — CHANGES_REQUESTED** [review]
See comments in code.

File: code/betawatch.py (hunk 1 above)
Comment (Andreas-Menzel) [reply code/betawatch.py]: done!

File: code/betawatch.py (hunk 2 above)
Comment (Andreas-Menzel) [reply code/betawatch.py:106]: done!

Comment (tizianerlenberg) [reply code/betawatch.py:106]: Thanks

File: code/betawatch.py (hunk 1 above)
Comment (tizianerlenberg) [reply code/betawatch.py]: Thanks

**Review by tizianerlenberg — APPROVED** [review]
Approved.

**Pull request merged**